src/snowcli/cli/common/snow_cli_global_context.py

```diff
@@ -0,0 +1,41 @@
+from copy import deepcopy
+from dataclasses import dataclass
+from typing import Callable
+
+
+@dataclass
+class LoggingAndExceptionHandlingContext:
+    enable_tracebacks: bool
+
+
+@dataclass
+class SnowCliGlobalContext:
+    logging_and_exception_handling: LoggingAndExceptionHandlingContext
```

Comment:
We will have some more fields soon? Because double wrapping boolean is overkill now

Reply:
I can agree to flatten the structure, however I don't think it's overkill. It introduces some convention from the very beginning.

Reply:
I think this is possible over-optimisation. I would +1 simpler, flat context. It will be easier to reason about what's in there.

Reply:
As we agreed to not use the solution from this PR, I made changes from this comment in new PR: https://github.com/Snowflake-Labs/snowcli/pull/192